### PR TITLE
Testing merge=union on CHANGELOG to avoid conflicts

### PR DESCRIPTION
**Pull request by jimmynotjim**
Testing merge=union on CHANGELOG to avoid conflicts

## Additions

- gitattributes file with merge rule for CHANGELOG

## Testing

- Let's merge some PRs :D

## Review

- @anselmbradford 
- @KimberlyMunoz 
- @sebworks 
- @Scotchester 

## Notes

- See https://github.com/cfpb/capital-framework/issues/304

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by kurtw** [comment]
**PLEASE LET THIS WORK**

**Comment by jimmynotjim** [comment]
@kurtw I know right? Half the time merging a PR is like Night at the Roxbury

giphy

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+CHANGELOG merge=union
```
Comment (marteki) [inline .gitattributes]: Add a slash at the start of the line? `/CHANGELOG`

**Comment by marteki** [comment]
This has worked so, so well for both retirement and college-costs. It's a whole new world, folks.

File: .gitattributes (hunk 1 above)
Comment (jimmynotjim) [inline .gitattributes]: ¯\_(ツ)_/¯ seems blog posts go both ways, I'm up to adding it for safety sake

**Pull request merged**